### DISCUSSION TITLE
How to remove DNS names from existing certificate

### DISCUSSION THREAD

**Discussion by brboyer** [polymind-inc/acmebot#419] [Q&A] [answered]
We have already certificate issued with DNS names we want to remove. We would like to re-issue a new cert with the same name , but with just one DNS entry vs the five that are on the current certificate. If we remove the certificate from the keyvault and try to re-issue the certificate with the same name we get and error "Problem finalizing order:: Order includes different number of names than CSR specifies".

Is there a way to generate a new CSR/Certificate with the same cert name ? Would we have to revert to using certbot in manual mode to modify the cert ?

**Comment by shibayan** [comment, marked as answer]
I think you can edit the Issuance Policy of the Key Vault Certificate and delete the unnecessary DNS names.
